openetl_utils/main_storage_class.py

```
@@ -0,0 +1,175 @@
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)